config/start_py_test.py

```diff
@@ -0,0 +1,30 @@
+# pylint: skip-file
+
+import argparse
```

Comment:
rename to reflect content - it is not a test, it is a check script, check_start_content